Package.swift

```diff
@@ -1,11 +1,12 @@
-// swift-tools-version: 5.6
+// swift-tools-version:5.9
```

Comment:
We need two different version of the package to support older versions of swift.